R/usepkg.R

```diff
@@ -10,8 +10,12 @@
 #' @export
 #'
 #' @examples
+#' 
 #' # packages shouldn't be installed during tests or examples according to CRAN. 
 #' # therefore, examples cannot be provided because CRAN now runs donttest examples.
+#' 
+#' usepkg('geodist', FALSE, 'http://cran.us.r-project.org')
```

Comment:
slight chance this will cause issues according to the comment above, but let's keep it and we'll deal with it later if the CRAN tests fail.